src/app/components/project/ProjectGroup.js

```diff
@@ -4,103 +4,106 @@ import { QueryRenderer, graphql } from 'react-relay/compat';
 import Relay from 'react-relay/classic';
 import { FormattedMessage } from 'react-intl';
 import FolderSpecialIcon from '@material-ui/icons/FolderSpecial';
+import ErrorBoundary from '../error/ErrorBoundary';
 import Search from '../search/Search';
 import { safelyParseJSON } from '../../helpers';
 import ProjectActions from '../drawer/Projects/ProjectActions';
 
 const ProjectGroup = ({ routeParams }) => (
-  <QueryRenderer
-    environment={Relay.Store}
-    query={graphql`
-      query ProjectGroupQuery($id: ID!) {
-        project_group(id: $id) {
-          id
-          dbid
-          title
-          description
-          team {
+  <ErrorBoundary component="ProjectGroup">
+    <QueryRenderer
+      environment={Relay.Store}
+      query={graphql`
+        query ProjectGroupQuery($id: ID!) {
+          project_group(id: $id) {
             id
-            slug
-            permissions
```

Comment:
I see you moved `slug` and `permissions` into a subquery here, just making sure that was as intended. I don't see any code changes made in response to this move, which is why I ask, but maybe I'm missing one.

Reply:
I didn't. They're under `team` all along. I guess it's the crazy diff visualization that may suggest it, but thanks for pointing!